packages/dmn-js-shared/test/spec/features/expression-languages/ExpressionLanguagesSpec.js

```diff
@@ -0,0 +1,151 @@
+import { bootstrap, getViewerJS } from '../../base/viewer/TestHelper';
+
+import ExpressionLanguagesModule from 'lib/features/expression-languages';
+
+
+const DEFAULT_OPTIONS = [{
+  label: 'FEEL',
+  value: 'feel'
+}, {
+  label: 'JUEL',
+  value: 'juel'
+}, {
+  label: 'JavaScript',
+  value: 'javascript'
+}, {
+  label: 'Groovy',
+  value: 'groovy'
+}, {
+  label: 'Python',
+  value: 'python'
+}, {
+  label: 'JRuby',
+  value: 'jruby'
+}];
+
+
+describe('ExpressionLanguages', function() {
+
+  describe('defaults', () => {
+
+    it('should set default editor expression language to JUEL', () => {
+
+      // given
+      const expressionLanguages = createExpressionLanguages();
+
+      // when
+      const defaultLanguage = expressionLanguages.getDefault('editor');
+
+      // then
+      expect(defaultLanguage).to.have.property('value', 'juel');
```

Comment:
Do we have any other properties here? Otherwise a simple `expect(x).to.eql({ some: 'object', other: 'key' })` does the job, too.

Reply:
No. I will change this accordingly.